pkg/ssh/manager.go

```diff
@@ -183,8 +206,14 @@ func (sm *StreamManager) UpdateRecords(ctx context.Context, _ uint64, records []
 	defer sm.mu.Unlock()
 	for _, record := range records {
 		if record.DeletedAt == nil {
+			var s session.Session
+			if err := record.Data.UnmarshalTo(&s); err != nil {
+				panic(err.Error())
```

Comment:
Probably better to log an error and continue

Reply:
Yeah probably a good idea